src/main/frontend/pages/TeamSpacePage.tsx

```
@@ -0,0 +1,1009 @@
+            { id: 'comment1-2', cardId: 'card1', userId: 'otherUser', userName: '박해커', text: '네, 거의 다 됐습니다!', createdAt: new Date(Date.now() - 1800000) },
+        ];
+         const demoCommentsUser3: KanbanComment[] = [
+            { id: 'comment3-1', cardId: 'card3', userId: currentUser.id, userName: currentUser.name || '김코딩', text: 'OAuth 부분에서 이슈가 있는데 같이 봐주실 수 있나요?', createdAt: new Date(Date.now() - 7200000) },
+        ];
+
+        const demoCards: KanbanCardType[] = [
+            {id: 'card1', title: '로그인 페이지 디자인', description: '사용자 인증 UI 구현', columnId: 'col1', order: 0, assigneeIds: [currentUser.id], comments: demoCommentsUser1, dueDate: new Date(Date.now() + 2 * 86400000)},
+            {id: 'card2', title: 'API 문서 작성', description: 'REST API 명세서 작성', columnId: 'col2', order: 0, assigneeIds: ['backend_dev_id'], dueDate: new Date(Date.now() + 5 * 86400000), comments: demoCommentsUser3, assigneeIds: [currentUser.id, 'backend_dev_id']},
+            {id: 'card3', columnId: 'col2', title: '로그인 기능 개발', description: 'OAuth 2.0 (Google, Kakao) 연동 및 자체 이메일/비밀번호 로그인 기능 구현. JWT 토큰 기반 인증.', order: 0, dueDate: new Date(Date.now() + 5 * 86400000), comments: demoCommentsUser3, assigneeIds: [currentUser.id, 'backend_dev_id']},
+            {id: 'card4', columnId: 'col3', title: '1차 QA 완료', description: '회원가입 및 로그인 플로우, 기본 팀 생성 기능에 대한 QA 완료됨.', order: 0, isApproved: true},
+            {id: 'card5', columnId: 'col3', title: '팀 공지사항 UI 개발', description: '팀 스페이스 내 공지사항 CRUD UI 개발 완료.', order: 1, isApproved: false},
```

Biome (1.9.4)

[error] This property value named assigneeIds is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named assigneeIds


(lint/suspicious/noDuplicateObjectKeys)